game.py

```diff
@@ -20,8 +20,34 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses  = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
 
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES  :
+
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+
+        if user_input in snowman_word :
+            print("You guessed a letter that's in the word!")
+            correct_letter_guess_statuses [user_input] =True
+
```

Comment:
```suggestion
```

Unnecessary blank line should be removed. 

Avoid adding unnecessary blank lines to your project because it makes your codebase longer and doesn't comply with the PEP8 style guide.

The guide says "Surround top-level function and class definitions with two blank lines." and "Extra blank lines may be used (sparingly) to separate groups of related functions."

Review the style guide recommendations on blank lines [here](https://peps.python.org/pep-0008/#blank-lines).